### PR TITLE
feat(api): add project notifications

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

File: sdk/event_v2.go [hunk 1]
```diff
@@ -82,6 +86,7 @@ type FullEventV2 struct {
 	Permission    string          `json:"permission,omitempty"`
 	Plugin        string          `json:"plugin,omitempty"`
 	GPGKey        string          `json:"gpg_key,omitempty"`
+	Notification  string          `json:"notification"`
```
Comment (yesnault) [inline sdk/event_v2.go:89]:
```suggestion
	Notification  string          `json:"notification,omitempty"`
```

**Comment by ovh-cds** [comment]
CDS Report build-cds#19836.0 ✘  
* Build
  * Build and Package All ✘  
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✘  
  * Test UI ✔

**Review by yesnault — CHANGES_REQUESTED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-cds#19837.0 ✔ 
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#19837.0 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

File: cli/cdsctl/experimental_project_notification.go [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+	},
+}
+
+func projectNotificationDeleteFunc(v cli.Values) error {
+	return client.ProjectNotificationDelete(context.Background(), v.GetString(_ProjectKey), v.GetString("name"))
+}
+
+var projectNotificationImportCmd = cli.Command{
+	Name:    "import",
+	Short:   "Import a notification on a project from a yaml file",
```
Comment (fsamin) [inline cli/cdsctl/experimental_project_notification.go:60]:
```suggestion
	Short:   "Import notification settings on a project from a yaml file",
```

File: cli/cdsctl/experimental_project_notification.go [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+	if n != nil && v.GetBool("force") {
+		content.ID = n.ID
+		return client.ProjectNotificationUpdate(context.Background(), v.GetString(_ProjectKey), &content)
+	}
+	return client.ProjectNotificationCreate(context.Background(), v.GetString(_ProjectKey), &content)
+}
+
+var projectNotificationExportCmd = cli.Command{
+	Name:    "export",
+	Short:   "Export a notification from a project to stdout",
```
Comment (fsamin) [inline cli/cdsctl/experimental_project_notification.go:102]:
```suggestion
	Short:   "Export notification settings from a project",
```

File: cli/cdsctl/experimental_project_notification.go [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+	Example: "cdsctl notification export MY-PROJECT MY-VCS-SERVER-NAME > file.yaml",
```
Comment (fsamin) [inline cli/cdsctl/experimental_project_notification.go:103]:
```suggestion
	Example: "cdsctl notification export MY-PROJECT MY-VCS-SERVER-NAME",
```

**Review by fsamin — CHANGES_REQUESTED** [review]

**Review by richardlt — APPROVED** [review]

**Review by yesnault — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-cds#19858.0 ✔ 
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-cds#19860.0 ✔ 
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#19860.0 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Review by fsamin — APPROVED** [review]

**Pull request merged**